src/vue/forms/WithdrawalForm.vue

```diff
@@ -277,9 +279,13 @@ export default {
       return this.form.asset.owner === Sdk.networkDetails.adminAccountId
     },
 
-    selectedAssetStep () {
-      // eslint-disable-next-line
-      return inputStepByDigitsCount(this.form.asset.trailingDigitsCount) || config.MIN_AMOUNT
+    trailingDigitsCount () {
+      return this.assets
+        .find(asset => asset.code === this.form.asset.code)
+        .trailingDigitsCount || config.DECIMAL_POINTS
+    },
+    minValue () {
```

Comment:
This computed do nothing. Remove it and in the 57 line use just `config.MINIMAL_NUMBER_INPUT_STEP`